easybuild/easyconfigs/t/tbb/tbb-2020.3-GCCcore-10.2.0.eb

```diff
@@ -14,6 +14,14 @@ checksums = ['ebc4f6aa47972daed1f7bf71d100ae5bf6931c2e3144cf299c8cc7d041dca2f3']
 builddependencies = [
     ('binutils', '2.35'),
     ('CMake', '3.18.4'),
+    ('SWIG', '4.0.2'),
 ]
+dependencies = [
+    ('Python', '3.8.6'),
```

Comment:
@zarybnicky The easyconfigs test suite is failing, because adding `Python` 3.8.6 as a dependency causes trouble in easyconfigs that (indirectly) depend on `tbb` but also depend on `Python` 2.7.18...

To avoid this, we would have to add a `versionsuffix` to `tbb`, which also requires renaming the easyconfig file.

Instead, we could simply *add* an `tbb-2020.3-GCCcore-10.2.0-Python-3.8.6.eb` easyconfig, which would avoid that issue.

We'll still be in a bit of a pickle with `epiScanpy` that depends on the `tbb` Python bindings though...

Another option could be to install the `tbb` Python bindings as a separate module, and naming that `tbb-python` or something...
Would that be feasible?